### PR TITLE
docs: add GraphQL support to readme

### PR DESCRIPTION
**Pull request by liuxiran**
### What this PR does / why we need it:

Add `Support filter route by graphql attributes` to readme
### Pre-submission checklist:

* [X] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [X] Have you modified the corresponding document?
* [X] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by wu-sheng — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
   - Support [TTL](docs/en/latest/admin-api.md#route)
   - [Support priority](docs/en/latest/router-radixtree.md#3-match-priority)
   - [Support Batch Http Requests](docs/en/latest/plugins/batch-requests.md)
+  - [Support filter route by graphql attributes](https://apisix.apache.org/docs/apisix/router-radixtree/#how-to-filter-route-by-graphql-attributes)
```
Comment (juzhiyuan) [inline README.md:102]:
```suggestion
  - [Support filter route by GraphQL attributes](https://apisix.apache.org/docs/apisix/router-radixtree/#how-to-filter-route-by-graphql-attributes)
```

**Review by juzhiyuan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (spacewander) [inline README.md:102]: "Support filter" should be `Support filtering`.
And the link should point to a file in this repo. The doc on the website is not the latest.

**Review by spacewander — CHANGES_REQUESTED** [review]

**Comment by spacewander** [comment]
Also, need to update the Chinese version

**Comment by liuxiran** [comment]
> Also, need to update the Chinese version

Got it, let me update it.

**Review by moonming — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (liuxiran) [reply README.md]: fixed, thanks.

I have a question, the docs on our website should be the same with the docs in this repo right? I'll check it later, if it is not the same, maybe  website need to update it.^^

File: README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
   - Support [TTL](docs/en/latest/admin-api.md#route)
   - [Support priority](docs/en/latest/router-radixtree.md#3-match-priority)
   - [Support Batch Http Requests](docs/en/latest/plugins/batch-requests.md)
-  - [Support filter route by GraphQL attributes](https://apisix.apache.org/docs/apisix/router-radixtree/#how-to-filter-route-by-graphql-attributes)
+  - [Support filtering route by GraphQL attributes](/docs/en/latest/router-radixtree.md#how-to-filter-route-by-graphql-attributes)
```
Comment (spacewander) [inline README.md:102]: Do we need a prefix "/" in the link?

Comment (liuxiran) [reply README.md]: my fault, updated ,thanks

File: docs/zh/latest/README.md [hunk 3]
```diff
@@ -103,6 +103,7 @@ A/B 测试、金丝雀发布(灰度发布)、蓝绿部署、限流限速、抵
   - 支持路由的[自动过期(TTL)](admin-api.md#route)
   - [支持路由的优先级](../../en/latest/router-radixtree.md#3-match-priority)
   - [支持批量 Http 请求](plugins/batch-requests.md)
+  - [支持通过GraphQL属性过滤路由](docs/en/latest/router-radixtree.md#how-to-filter-route-by-graphql-attributes)
```
Comment (spacewander) [inline docs/zh/latest/README.md:106]:
```suggestion
  - [支持通过GraphQL属性过滤路由](../../en/latest/router-radixtree.md#how-to-filter-route-by-graphql-attributes)
```

**Review by spacewander — APPROVED** [review]

**Pull request merged**